test_pull_requests

```diff
@@ -1650,7 +1805,12 @@ elsif mark_test_success_pull_id
 elsif mark_testonlyextended_success_pull_id
   @api_client.mark_test_success(mark_testonlyextended_success_pull_id, pull_id_repo, Properties['settings']['testonlyextended_settings'])
 elsif local_merge_pull_id
-  @api_client.local_merge_pull_request(local_merge_pull_id, pull_id_repo)
+  if ENV['MERGE'].to_s.downcase == 'true'
```

Comment:
Correct.  If you look through the rest of the code you won't see any references to it.  It's entirely a concept that exists outside in your jenkins environment.